CHANGELOG.md

```diff
@@ -5,39 +5,58 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 3.7.0
+
+### Added
+
+* Add support for CronJob, Job, PersistentVolumeClaim, PersistentVolume kube-state-metrics
+
+### Changed
+
+* Fix an issue where when the Kubelet becomes temporarily unavailable the agent fails: https://github.com/newrelic/nri-kubernetes/pull/633
+* Update e2e testing chart templates
+* Updated dependencies
+
+**Full Changelog**: https://github.com/newrelic/nri-kubernetes/compare/v3.6.0...v3.7.0
+
```

Comment:
I added lines 8-20. All other changes below are thanks to my markdown autoformatter. :/